src/activation.jl

```diff
@@ -45,7 +45,7 @@ relu(x::Real) = max(zero(x), x)
 
 
 """
-    leakyrelu(x) = max(0.01x, x)
+    leakyrelu(x, a = 0.01) = max(ax, x)
```

Comment:
I suggest using no whitespace in keyword arguments, as this seems to be the stylistic standard in julia
```suggestion
    leakyrelu(x, a=0.01) = max(a*x, x)
```

Reply:
But those aren't keywords. I thought the style was something like `leakyrelu(x, [a = 0.01]) = max(ax, x)`.

Reply:
> Optional arguments should be represented with their default values (i.e. f(x, y=1)) when possible, following the actual Julia syntax. Optional arguments which do not have a default value should be put in brackets (i.e. f(x[, y]) and f(x[, y[, z]])). 

https://docs.julialang.org/en/v1/manual/documentation/index.html#Documentation-1

Since our functions have default values, I would follow the style suggested by @CarloLucibello